src/app/models/hris/EmployeeProfile/employeeProfileBankingInformation.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { EmployeeBanking } from "../employee-banking.interface";
+import { EmployeeData } from "../employee-data.interface";
+import { EmployeeProfileSalary } from "./employeeProfileSalaryDetails.interface";
+
+export interface employeeProfileBanking {
+    employeeSalary : EmployeeProfileSalary,
```

Comment:
This is not needed anymore I took it out of the back-end "employeesalary"

Reply:
updated